modules/ROOT/pages/policy-mule3-ip-whitelist.adoc

```diff
@@ -22,41 +22,41 @@ You can set up access in the following ways:
 The list of any applied policies appears.
 +
 . In Apply New Policy.
-. In Select Policy, choose IP Whitelist, and click Configure Policy.
+. In Select Policy, choose IP Allowlist, and click Configure Policy.
 . Specify an address or a range or addresses as follows:
 +
 * Define a specific IP address. For example, 192.168.1.1.
 * Define a subset of addresses by identifying a subnet mask, such as 192.168.3.1/30. This includes the consolidated range: 192.168.3.0 - 192.168.3.3.
 * Define a whole range of IP addresses by stating the relevant octets of the IP address you want to permit. For example, setting `192.168` will include IP addresses from 192.168.0.0 to 192.168.255.255.
 
-To whitelist a list of IP addresses:
+To allowlist a list of IP addresses:
 
-. Enter one or more IP addresses in *Whitelist*.
+. Enter one or more IP addresses in *Allowlist*.
 +
-image::ip-whitelist-051a0.png[height=353,width=377]
+image::ip-whitelistlist-051a0.png[height=353,width=377]
```

Comment:
Might want to double check this image name

Reply:
Awesome catch! Thanks Sathya!